# Sensor Hub state

#Ultrasonic Sensors:
uint8 UltraSonicSensorLeft_cm
uint8 UltraSonicSensorMiddle_cm
uint8 UltraSonicSensorRight_cm

#Radar Sensor:
uint8 RadarSensorAktivity

#Temperature Sensor:
uint8 Temperature_celsius